docs/docs/cli-governance.md

```diff
@@ -0,0 +1,68 @@
+# Governance Proposals
+
+In order to change any attribute of a network, a governance proposal must be submitted. This could be a simple poll, a software update or a governing parameter change. 
+
+### Parameter change
+This is an example of the process in which network parameters may be changed through the use of a governance proposal.
+
+The values within this code can be changed in order to alter the minimum deposited fund threshold for a proposal to enter the voting phase - alongside the length of the deposit stage in which the minimum deposit threshold must be met.
+```
+# A JSON file containing the following code should be created to instantiate the proposal.
+# The two variables of interest are the "amount" which is set from 10000000stake to 1000stake
+# and the "max_deposit_period" which is changed from the default value to 7200000000000
+# equal to 2 hours, instead of the standard 2 days (in nanoseconds).
+
+{
+  "title": "Deposit Value Proposal",
+  "description": "Update min proposal threshold and deposit period",
+  "changes": [
+    {
+      "subspace": "deposit_params",
+      "key": "min_deposit",
+      "value": [
+        {
+          "denom":"stake",
+          "amount":"1000"
+        }
+      ]
+    },
+    {
+      "subspace": "deposit_params",
+      "key": "max_deposit_period",
+      "value": "7200000000000"
+    }
+  ]
+}
+```
+```
+# Create initial proposal by uploading the JSON file
+# this is signed by a key 'proposer' that provides a portion of the current threshold deposit
+fetchd tx gov submit-proposal --proposal ~/json_path/proposal.json --deposit <deposit_value> --from proposer
+
+# In order to later refer to this proposal, the proposal_id can be determined
+fetchd query gov proposals
+```
+
+### Proposal voting and querying
+After the deposit period has passed, there are two outcomes: either the current minimum threshold is met, or the value is not met and the funds are burned. In the first case this proposal is submitted and then voted on, returning a tally at the end of the voting period.
```

Comment:
`"the funds are burned" ` Wow this sounds scary. Is there any reference we could link here or add some description on how to define an appropriate amount for the proposal value ?

Reply:
Right so the funds aren't burned in all cases. From the link, you added (https://docs.cosmos.network/master/modules/gov/01_concepts.html#deposit-refund-and-burn), that burn only happens if the proposal is vetoed by a majority. Accepted or rejected proposals get their deposit refunded, which is a lot less scary. @Josumner